lib/udf_findhotel_utils.rb

```diff
@@ -297,29 +297,31 @@ def get_bid_match_type(items):
           type:        :function,
           name:        :make_gha_click_batch_id,
           description: "Returns a unique identifier for a batch of clicks reported by Google Hotel Ads.",
-          params:      "click_type varchar(max), date_type varchar(max), google_site varchar(max), country varchar(max), device varchar(max)",
+          params:      "date_type varchar(max), google_site varchar(max), country varchar(max), device varchar(max), hotel_id varchar(max), checkin varchar(max), los bigint",
           return_type: "varchar(max)",
           body:        %~
             import hashlib
             import json
 
             key = {
-                "click_type": click_type,
```

Comment:
I removed this parameters because the value reported and the one we get from the tracking never match, also the value seems to be always the same, thus it would not have any effect in the batch_id.